pkg/work/spoke/controllers/manifestcontroller/manifestwork_controller.go

```diff
@@ -138,7 +137,7 @@ func (m *ManifestWorkController) sync(ctx context.Context, controllerContext fac
 	}
 	newAppliedManifestWork := appliedManifestWork.DeepCopy()
 
-	var requeueTime = MaxRequeueDuration
+	var requeueTime = ResyncInterval
```

Comment:
ResyncInterval is also set in withResync. Should these two to be set the same?

Reply:
Do you mean `ResyncEvery`? It has already used the `ResyncInterval` although I think it is useless for the `ManifestWorkAgent` controller.
```go
WithSync(controller.sync).ResyncEvery(ResyncInterval).ToController("ManifestWorkAgent", recorder)
```

Reply:
can we add some comment for ResyncInterval, given it is used at multiple locations.

Reply:
Done.